README.md

```diff
@@ -218,7 +218,7 @@ chrono documentation: [https://docs.rs/chrono-tz/latest/chrono_tz/enum.Tz.html](
 datetime        "#[fg=#6C7086,bold] {format} "
 
 # format of the date. e.g. if you'd like to only show the time
-datetime_format "%A, %d %b %Y %H:%M"
+datetime_format "%H:%M"
```

Comment:
Can we keep the date here and change the comment? I'd like to provide more information, such that a user can filter out irrelevant information. If we remove these, a user must search for the other ones, which involves more effort.

Reply:
Thank you for your feedback! I've reverted the changes to the `datetime_format`. Additionally, I've included a reference to the format specifiers in the chrono documentation. Given that I was not initially familiar with the syntax myself, I thought it might be beneficial to include this extra documentation for anyone else who might be in the same situation. Does this seem like too much information?

If there are any further adjustments needed, please let me know!